### PR TITLE
Revise scoring text

### PR DESCRIPTION
**Pull request by sigurdm**
Fixes https://github.com/dart-lang/pub-dev/issues/5419

### CODE REVIEW

File: doc/help-scoring.md [hunk 1]
```diff
@@ -74,31 +76,54 @@ This category measures if a package has documentation in two areas:
...
+* android
+* ios
+* web
 
- - **Dart packages**: For packages that depend on just the Dart SDK, platform
-   support is inferred by looking at the imports of [Dart core
-   libraries](https://dart.dev/guides/libraries); the majority of these are
-   multi-platform, but [as listed](https://dart.dev/guides/libraries) a few
-   support just the Dart native or Dart web platforms.
+And the two SDKs
```
Comment (isoos) [inline doc/help-scoring.md:96]: `And the two SDKs:` (`:` at the end)

**Review by isoos — APPROVED** [review]

File: doc/help-scoring.md [hunk 2]
```diff
@@ -29,17 +29,19 @@ option.
 ## Popularity
 
 Popularity measures the number of apps that depend on a package over the past 60
-days. We show this on a normalized scale from 100% (the most used package) to 0%
-(the least used package), but are investigating if we can provide absolute usage
-counts in a future version.
+days. We show this as a percentile from 100% (among the top 1% of packages) to
```
Comment (mit-mit) [inline doc/help-scoring.md:32]: could we s/among the top 1% of packages/amount the top 1% most used packages/ ?

File: doc/help-scoring.md [hunk 3]
```diff
@@ -29,17 +29,19 @@ option.
...
+0% (the least used package), but are investigating if we can provide absolute
```
Comment (mit-mit) [inline doc/help-scoring.md:33]: suggest s/, but are/. We are/

File: doc/help-scoring.md [hunk 4]
```diff
@@ -74,31 +76,54 @@ This category measures if a package has documentation in two areas:
...
 [desktop](https://flutter.dev/desktop) targets.
 
-Pub.dev uses two different algorithms for determining which platforms are supported:
+pub.dev knows about the following platforms:
 
- - **Flutter packages**: For packages that depend on the Flutter SDK, platform
-   support is determined by looking at the [`platforms`
-   tag](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms)
-   in the `pubspec.yaml` file.
+* windows
```
Comment (mit-mit) [inline doc/help-scoring.md:89]: I think the proper capitalisation for these is:

Windows
Linux
macOS
Android
iOS
Web

**Review by mit-mit — COMMENTED** [review]
lgtm, just some nits

File: doc/help-scoring.md (hunk 2 above)
Comment (sigurdm) [reply doc/help-scoring.md]: Yeah that's more precise!

File: doc/help-scoring.md (hunk 3 above)
Comment (sigurdm) [reply doc/help-scoring.md]: Done

File: doc/help-scoring.md (hunk 4 above)
Comment (sigurdm) [reply doc/help-scoring.md]: Yeah - I kind-a wanted to give the keywords one could put in the pubspec, but that should not go here.

Fixed.

**Pull request merged**